public/api-documentation/swagger.json

```diff
@@ -262,6 +262,46 @@
         }
       }
     },
+    "/manifests/{id}/cromerr_submit_transaction": {
```

Comment:
:+1:
CROMERR is part of it, not quite all, but it's better to have names that make sense.